op-service/signer/transaction_args.go

```diff
@@ -190,3 +181,15 @@ func (args *TransactionArgs) ToTransactionData() (types.TxData, error) {
 	}
 	return data, nil
 }
+
+// mustUint256FromBig converts a *big.Int to a *uint256.Int, panicking on overflow
+func mustUint256FromBig(b *big.Int) *uint256.Int {
```

Comment:
There is `uint256.MustFromBig` that does the same thing

Reply:
Ah good call. Fixed here https://github.com/ethereum-optimism/optimism/pull/14489/commits/ec7af02877135c4b8eca8047d28a10fc45c9dfc3